src/features/editor/ledgers/EditorPannel.tsx

```diff
@@ -0,0 +1,32 @@
+import * as React from "react";
+import { TextIcon } from "../../../common/components/TextIcon";
+import { Tool } from "../../../common/components/Tool";
+import { useDraggableHandle } from "../../../common/hooks/useDraggableHandle";
+import { useToggle } from "../../../common/hooks/useToggle";
+
+export function EditorPannel() {
+  const [open, toggleOpen] = useToggle(true);
+  const [dragState, el, handle] = useDraggableHandle("vertical", "end");
+
+  return open ? (
+    <div
+      ref={el}
+      className="editor-pannel"
+      style={open ? { width: dragState.currentSize + "px" } : {}}
+    >
+      <Tool name="Browse" close={toggleOpen}>
+        <p>This is the browse tool content</p>
+      </Tool>
+      {handle}
+    </div>
+  ) : (
+    <aside className="left collapsed" onClick={toggleOpen}>
+      <ul>
+        <li className="left">
+          <TextIcon name="manage_search" />
+          <span className="left">Browse</span>
```

Comment:
Got some feedback from Ian @alpian previously that when the browseSideBar or Result panel is collapsed, we should only click the "Browse" or "Results" area to toggle instead of the whole collapsed bar.

Reply:
That's a good point. I confirmed with JR and his preference is that only the icon should be clickable. I'll change that in my upcoming CR, if that's ok?!

Reply:
Sure thanks!